src/assets/styles/scss/components/landing/hero/title-area/_title-area.scss

```diff
@@ -0,0 +1,138 @@
+/* HeroTitleAreaViewContainer */
+
+.hero-title-area-view-container {
+  position: absolute;
```

Comment:
let's nest styles to improve readability. we shouldn't need comments to tell what the style is for!